### DISCUSSION TITLE
An error occurs when starting the Chinese document site preview

### DISCUSSION THREAD

**Discussion by Kick156** [apache/streampark#3442] [Q&A] [answered]
I modified the article in Idea, and I want to start the page to browse, and the following error occurs。

Compiled with problems:
×
ERROR in ./i18n/zh-CN/docusaurus-plugin-content-docs/current/connector/4-doris.md
Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
Error: MDX compilation failed for file "D:\incubator-streampark-website-dev\incubator-streampark-website-dev\i18n\zh-CN\docusaurus-plugin-content-docs\current\connector\4-doris.md"
Cause: Could not parse expression with acorn
Details:
{
  "cause": {
    "pos": 494,
    "loc": {
      "line": 16,
      "column": 66
    }
  },
  "column": 67,
  "message": "Could not parse expression with acorn",
  "line": 16,
  "name": "16:67",
  "place": {
    "line": 16,
    "column": 67,
    "offset": 494
  },
  "reason": "Could not parse expression with acorn",
  "ruleId": "acorn",
  "source": "micromark-extension-mdx-expression",
  "url": "https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#could-not-parse-expression-with-acorn"
}

    at Object.mdxLoader (D:\incubator-streampark-website-dev\incubator-streampark-website-dev\node_modules\@docusaurus\mdx-loader\lib\loader.js:126:25)
ERROR in ./i18n/zh-CN/docusaurus-plugin-content-docs-community/current/submit_guide/submit-code.md
Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
Error: MDX compilation failed for file "D:\incubator-streampark-website-dev\incubator-streampark-website-dev\i18n\zh-CN\docusaurus-plugin-content-docs-community\current\submit_guide\submit-code.md"
Cause: Unexpected character `/` (U+002F) before local name, expected a character that can start a name, such as a letter, `$`, or `_` (note: to create a link in MDX, use `[text](url)`)
Details:
{
  "column": 18,
  "message": "Unexpected character `/` (U+002F) before local name, expected a character that can start a name, such as a letter, `$`, or `_` (note: to create a link in MDX, use `[text](url)`)",
  "line": 24,
  "name": "24:18",
  "place": {
    "line": 24,
    "column": 18,
    "offset": 914,
    "_index": 0,
    "_bufferIndex": 15
  },
  "reason": "Unexpected character `/` (U+002F) before local name, expected a character that can start a name, such as a letter, `$`, or `_` (note: to create a link in MDX, use `[text](url)`)",
  "ruleId": "unexpected-character",
  "source": "micromark-extension-mdx-jsx",
  "url": "https://github.com/micromark/micromark-extension-mdx-jsx#unexpected-character-at-expected-expect"
}

    at Object.mdxLoader (D:\incubator-streampark-website-dev\incubator-streampark-website-dev\node_modules\@docusaurus\mdx-loader\lib\loader.js:126:25)

I deleted these two error-related articles directly, but they still show up after restarting

![image](https://github.com/apache/incubator-streampark/assets/107044617/5074e938-8e8a-4c7c-9f4a-78edf2585e0d)

Does anyone know how to fix it

**Comment by caicancai** [comment]
@wangsizhu0504 If you have time, can you help me take a look?

**Comment by wangsizhu0504** [comment, marked as answer]
@Kick156  Hi, is it convenient for you to give me your specific changes? It seems to be a problem with md format conversion! If you introduce a link in the md file, please use the [text] (url) format of md.

**Comment by Kick156** [comment]
I didn't revise the two articles above that involved errors, and I haven't touched the documentation for a long time.
A few days ago, when I finished pulling the code of the project and was about to submit the article of ziroom to the project blog, I could not preview the official website, and it produced the following error:
![image](https://github.com/apache/incubator-streampark/assets/107044617/89c41ea9-4562-416d-b6a0-8cc2f91d2f32)
Here's my node version：
![image](https://github.com/apache/incubator-streampark/assets/107044617/44fe1f51-a508-41ae-a629-bba70875967f)
After troubleshooting, the dependencies in my package.json file are different from the updated project dependencies。
I need the following configuration to launch the web preview page。
![image](https://github.com/apache/incubator-streampark/assets/107044617/9507a500-7e62-47ae-9389-6c70365ea912)
When I start the page, the above two error messages appear, and the page I modified can be previewed normally.

**Reply by Kick156** [reply]
@wangsizhu0504

**Comment by caicancai** [comment]
@wangsizhu0504 If you have time, could you please answer this question? Thank you.
